contribute-docs/content-types/tutorials.md

```diff
@@ -0,0 +1,99 @@
+- An **overview:** Explain what the tutorial teaches, who it's for, and what users will be able to do by the end. Include:
+  - A brief description of what users will learn
+  - The intended audience and their expected skill level
+  - Learning objectives as a bulleted list
+- A **before you begin** section: List all prerequisites including:
+  - Required prior knowledge or skills
+  - Software, hardware, or access requirements
+  - Data sets or environments to set up
+  - Estimated time to complete (optional but helpful)
+- **Instructional steps:** Organize the tutorial into logical sections, each with a descriptive heading. Use numbered steps that begin with imperative verbs.
+  :::{tip}
+  Use the [stepper component](https://elastic.github.io/docs-builder/syntax/stepper/) for visual flow, or add subheadings to break complex steps into subsections.
+  :::
+- **Checkpoints and results:** After significant steps, show users what they should see or what state their system should be in.
+- **[Code annotations](https://elastic.github.io/docs-builder/syntax/code/#code-callouts):** Explain important lines within code blocks to help users understand the code. Annotations can help reduce the need for extra text in the body of the tutorial, keeping the tutorial concise and focused. This makes it easier for a reader to simply run through the steps in a hurry.
+- **Next steps:** Suggest follow-up tutorials, related features to explore, or ways to expand on what they built.
+- **Related pages:** Links to related documentation such as conceptual topics, reference material, how-to guides, or troubleshooting resources.
+
+### Optional elements
+
+Include the following when they add value:
+
+- **[Screenshots](https://elastic.github.io/docs-builder/syntax/images/#screenshots):** Add visual aids for UI-based steps when they improve clarity. Use screenshots sparingly as they require maintenance.
+- **Explanatory callouts:** Use [admonitions](https://elastic.github.io/docs-builder/syntax/admonitions/) to provide extra context, tips, or explanations without interrupting the main flow.
+- **Troubleshooting tips:** Mention common issues learners might encounter and how to resolve them.
+- **Time estimates:** Indicate how long each major section or the overall tutorial takes to complete.
+- A **summary:** Recap what users learned and accomplished in the tutorial. Reinforce the key learning objectives.
+% I've made the summary optional, because if the overview already lists learning objectives clearly, a summary that just repeats them feels redundant. The tutorial's value is in the journey, not bookending it with the same information. 🤺
+
+## Template
+
+To get started on writing your tutorial, use the [template](https://github.com/elastic/docs-content/blob/main/contribute-docs/content-types/_snippets/templates/tutorial-template.md).
+
+## Examples
+
+Here are some examples of well-structured tutorials in the Elastic documentation:
```

Comment:
Zero attachments to these specifically, and we wouldn't want two ESQL examples here, but they illustrate the two different flavors I've tried to tease out here (feature-focused versus scenario-driven tutorials) and they hew pretty close to the best practices.